src/sources/redis/mod.rs

```diff
@@ -108,16 +108,19 @@ pub struct RedisSourceConfig {
     /// The Redis URL to connect to.
     ///
     /// The URL must take the form of `protocol://server:port/db` where the `protocol` can either be `redis` or `rediss` for connections secured via TLS.
+    #[configurable(metadata(docs::examples = "redis://127.0.0.1:6379/0"))]
     url: String,
 
     /// The Redis key to read messages from.
+    #[configurable(metadata(docs::examples = "vector"))]
     key: String,
 
     /// Sets the name of the log field to use to add the key to each event.
     ///
     /// The value will be the Redis key that the event was read from.
     ///
     /// By default, this is not set and the field will not be automatically added.
+    #[configurable(metadata(docs::examples = "redis_key"))]
```

Comment:
🤔  Is it just me, or is the combination of the doc comments for these two members, confusing?

Is the value taken from "key" here, the value referred to by the comment in "redis_key" ?

Reply:
No, `redis_key` determines what event path data is added - `key` is the Redis key/value setup.

It's a difficult naming situation.